workflows/check_commit_message_for_tickets.yml

```diff
@@ -0,0 +1,92 @@
+name: Check Commit Message
+
+on:
+  pull_request:
+    types: [opened, synchronize, edited, reopened]
+
+jobs:
+  check-commit-message:
+    runs-on: ubuntu-latest
+    env:
+      REGEX_PATTERN: '^(?i)(ref|refs|reference|references|res|resolve|resolves)[ \t]*:[ \t]*(gh-|\#|\!|[A-Za-z]+-)\d+\s*$'
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Install jq
+        run: sudo apt-get install jq
+
+      - name: Fetch PR data
+        id: pr_data
+        run: |
+          PR_DATA=$(curl -s -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
+                    "https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}")
+          echo "PR_DATA<<EOF" >> $GITHUB_ENV
+          echo "$PR_DATA" >> $GITHUB_ENV
+          echo "EOF" >> $GITHUB_ENV
+
+      - name: Check PR body
+        run: |
```

Comment:
Do you know about `gh pr view <number> --json ... --jq ...`?

e.g. this could be done by doing:
```
gh pr view ${{ github.event.pull_request.number }} --json body --jq '.body | test("^(?i)(ref|refs|reference|references|res|resolve|resolves)[ \\t]*:[ \\t]*(gh-|\\#|\\!|[A-Za-z]+-)\\d+\\s*$")'
```

Reply:
I do, but I tried to keep this to as few dependencies as possible.